docs/reference/index-modules.asciidoc

```diff
@@ -83,6 +83,11 @@ WARNING: Expert only. Checking shards may take a lot of time on large indices.
     than the `index.number_of_shards` unless the `index.number_of_shards` value is also 1.
     See <<routing-index-partition>> for more details about how this setting is used.
 
+[[load-fixed-bitset-filters-eagerly]] `index.load_fixed_bitset_filters_eagerly`::
+
+    Indicates where <<query-filter-context, cached filters>> are pre-loaded for
```

Comment:
did you mean s/where/whether/?

Reply:
🤦‍♂️  I did mean 'whether.' Thanks for catching that @jpountz. Fixed with c8f5200.